src/Tribe/Template.php

```diff
@@ -613,4 +613,55 @@ public function template( $name, $context = array(), $echo = true ) {
 
 		return $html;
 	}
+
+	/**
+	 * Sets a number of values at the same time.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $values An associative key/value array of the values to set.
+	 * @param bool  $is_local Whether to set the values as global or local; defaults to local as the `set` method does.
+	 *
+	 * @see Tribe__Template::set()
+	 */
+	public function set_values( array $values = [], $is_local = true ) {
+		foreach ( $values as $key => $value ) {
+			$this->set( $key, $value, $is_local );
+		}
+	}
+
+	/**
+	 * Returns the Template global context.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative key/value array of the Template global context.
+	 */
+	public function get_global_values() {
+		return $this->global;
+	}
+
+	/**
+	 * Returns the Template local context.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative key/value array of the Template local context.
+	 */
+	public function get_local_values() {
+		return $this->context;
+	}
+
+	/**
+	 * Returns the Template global and local context values.
+	 *
+	 * Local values will override the template global context values.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative key/value array of the Template global and local context.
+	 */
+	public function get_values() {
```

Comment:
We need to make sure we are using this method on the `$this->get()`

Reply:
I've modified this method to fetch local values using `get_local_values` and to fetch global values using `get_global_values`.
Since we're applying the `tribe_template_context_get` filter right after that I feel we've got enough filtering.